src/i18n-keysets/component.collections-structure/en.json

```diff
@@ -18,7 +18,10 @@
   "label_dashboard": "dashboard",
   "label_dataset": "dataset",
   "label_delete-collection": "Delete collection",
+  "label_delete-collections-for-delete": "Collections to be deleted:",
```

Comment:
`Collections to be deleted:` -> `Collections to be deleted`
`Workbooks to be deleted:` -> `Workbooks to be deleted`

the same in `ru` locale.

<img width="537" alt="Screenshot 2024-06-23 at 19 18 48" src="https://github.com/datalens-tech/datalens-ui/assets/29424746/e01413ac-8c66-45b0-b94a-edd641a83e03">

Reply:
probably is was happed after pull-ups of translations, fixed it

Reply:
remove `:` from i18n keys plz, and add it to markup
it isn't connected with translation